contracts/StakeLocker.sol

```diff
@@ -62,9 +62,11 @@ contract StakeLocker is StakeLockerFDT, Pausable {
                2. The Pool is in Initialized or Deactivated state.
     */
     modifier canUnstake() {
+        IPool poolConrtact = IPool(pool);
```

Comment:
Lets just use `_pool` here, thats the pattern we've been using with the `IMapleGlobals` interface